reports/reporting/release_summary.py

```diff
@@ -67,10 +68,54 @@ def run(task_id, release_id):
     return counts
 
 
+def publish(release_id):
+    """
+    Update the study and release summary rows with the new version number and
+    set state to published
+
+    :param release_id: The kf_id of the release to update
+    """
+    endpoint_url = current_app.config['DYNAMO_ENDPOINT']
+    db = boto3.resource('dynamodb', endpoint_url=endpoint_url)
+    release_table = db.Table(current_app.config['RELEASE_SUMMARY_TABLE'])
+    study_table = db.Table(current_app.config['STUDY_SUMMARY_TABLE'])
+
+    studies, version, state = get_studies(release_id)
+
+    # Update study summary rows
+    for study_id in studies:
+        st = study_table.update_item(
+            Key={'release_id': release_id, 'study_id': study_id},
+            UpdateExpression='SET #st= :new, #vr= :next',
+            ExpressionAttributeNames={
+                '#st': 'state',
+                '#vr': 'version'
+            },
+            ExpressionAttributeValues={
+                ':new': 'published',
```

Comment:
all values of state column are `published` unless we run reports for `staged` state?

Reply:
Yeah, tasks can only be `staged` or `published` in the reports service, technically. This will only be run when a task is request for `publish`.